nnbuilder/src/main/kotlin/org/hse/nnbuilder/Layer.java

```diff
@@ -0,0 +1,84 @@
+package org.hse.nnbuilder;
+
+import java.io.Serializable;
+import org.json.JSONObject;
+
+class Layer implements Serializable {
+    enum LayerType {
+        InputCell,
+        BackfedInputCell,
+        NoisyInputCell,
+        HiddenCell,
+        ProbablisticHiddenCell,
+        SpikingHiddenCell,
+        CapculeCell,
+        OutputCell,
+        MatchInputOutputCell,
+        RecurrentCell,
+        MemoryCell,
+        GatedMemoryCell,
+        Kernel,
+        ConvolutionalOrPool
+    }
+
+    /**
+     * https://towardsdatascience.com/activation-functions-neural-networks-1cbd9f8d91d6
+     * Site with most used Activation functions
+     */
+    enum ActivationFunction {
+        None,
+        Linear,
+        Sigmoid,
+        Tanh,
+        ReLU,
+        LeakyReLU,
+        Max,
+        BinaryStep,
+        Gaussian
+    }
+
+    /* Layer type */
+    final LayerType lType;
+    /* Function for activation */
+    private ActivationFunction activationFunction;
+    /* List of neurons on this layer */
+    private int neurons;
+
+    Layer(ActivationFunction f, int n, LayerType t) {
+        lType = t;
+        activationFunction = f;
+        neurons = n;
+    }
+
+    Layer(int i, LayerType t) {
+        this(ActivationFunction.None, i, t);
+    }
+
+    Layer(LayerType t) {
+        this(ActivationFunction.None, 1, t);
+    }
+
+    /**
+     * @return json string with full description of Layer
+     */
+    JSONObject getLayerInformation() {
```

Comment:
-> `serialize` / `serializeToJson`, name `information` doesn't explain the purpose